src/actions/index.js

```diff
@@ -242,7 +242,15 @@ export function notifyObservers(payload) {
         );
         const queuedObservers = [];
         outputObservers.forEach(function filterObservers(outputIdAndProp) {
```

Comment:
I found the variable name `outputIdAndProp` confusing when reading this, since it can now refer to stuff like `[output1.children:output2.n_clicks]`